docs/source/technical_tutorials/access/external_access_from_hub.ipynb

```diff
@@ -0,0 +1,1310 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    matches = fs.glob(pattern)
</pre><p>Please don't scan the bucket for files, just hard code an example like the example above, or does the filename keep changing? Reason, we want users to rely on STAC and CMR to find files not bucket listing which won't always work depending on the DAAC and Collection.</p><p> </p><p>It also adds a lot of complication to the code example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/535/discussion/#comment-2600614982'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>